projects/Mallard/src/helpers/diagnostics.ts

```diff
@@ -74,8 +74,8 @@ Subscriber ID: ${isCAS(authStatus) && casCode}
 
 const openSupportMailto = (text: string, releaseURL: string, body?: string) => {
     const email = Platform.select({
-        ios: isInBeta() ? releaseURL : IOS_BETA_EMAIL,
-        android: isInBeta() ? releaseURL : ANDROID_BETA_EMAIL,
+        ios: (isInBeta() && releaseURL) || IOS_BETA_EMAIL,
```

Comment:
```suggestion
        ios: isInBeta() ? IOS_BETA_EMAIL : releaseURL,
```

Reply:
I don't have the original requirements, so will go with your suggestions.